src/Microsoft.AspNetCore.HttpsPolicy/HttpsPolicyBuilderExtensions.cs

```diff
@@ -65,5 +46,9 @@ public static IApplicationBuilder UseHttpsPolicy(this IApplicationBuilder app, H
 
             return app;
         }
+
+        public static IApplicationBuilder UseHttpsPolicy(this IApplicationBuilder app, bool enableHsts)
+        {
```

Comment:
Why empty? Do we need a comment explaining it?

Reply:
Sorry redesign for this. I need to do a bit of work still!

Reply:
Oh, OK, sorry didn't notice the [WIP] in the title.